CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+1.1.0 (2021-05-21)
+-------------------
+- Add location BigQuery configuration
+- Fix for reserved table names
+- Fix for bad decimals
+- Improved logging of bad data
+- Refactor code to make it more similar to other taps
+
+1.0.2 (2020-07-16)
```

Comment:
is this version exists? I can't see it in master or in pypi

Reply:
oh not at all, I was using that in our internal repo. Good catch! I just removed it

I have also named the new version we could generate after merging this as 1.1.0 but let me know if you are following another standard